src/components/common/form/MyInfoEditForm.jsx

```diff
@@ -7,18 +7,14 @@ import { getUserInfo } from "@/services/mypage";
 import { getChangeNickname, getChangePassword } from "@/services/mypage";
 import Swal from "sweetalert2";
 import routes from "@/routes";
-import { passwordCheck } from "@/utils/regex";
+import { passwordCheck, passwordReCheck } from "@/utils/regex";
 import MyBtn from "@/components/mypage/MyBtn";
 import { styled } from "styled-components";
 import { useQuery, useMutation } from "@tanstack/react-query";
 import { nicknameDoubleCheck } from "@/services/user";
 import { useNavigate } from "react-router-dom";
+import { login } from "../../../services/user";
```

Comment:
path alias 적용해놔서 그거 사용하면 될 것 같아요!!
```suggestion
import { login } from "@/services/user";
```